src/figuresClasses.ts

```diff
@@ -2,10 +1,98 @@
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle';
+
+  color: 'red' | 'green' | 'blue';
```

Comment:
The `color` property should be defined as `color: 'red' | 'green' | 'blue';` in the class to match the `Figure` interface.